test/node.js

```diff
@@ -37,6 +41,94 @@ test('node pass on nodeOptions', async t => {
 	t.is(stdout, 'foo');
 });
 
+test.serial(
+	'node removes --inspect from nodeOptions when defined by parent process',
+	async t => {
+		const originalArgv = process.execArgv;
+		process.execArgv = ['--inspect', '-e'];
+		const {stdout, stderr} = await execa.node('console.log("foo")', {
+			stdout: 'pipe'
+		});
+		process.execArgv = originalArgv;
+
+		t.is(stdout, 'foo');
+		t.is(stderr, '');
+	}
+);
+
+test.serial(
+	'node removes --inspect=9222 from nodeOptions when defined by parent process',
+	async t => {
+		const originalArgv = process.execArgv;
+		process.execArgv = ['--inspect=9222', '-e'];
+		const {stdout, stderr} = await execa.node('console.log("foo")', {
+			stdout: 'pipe'
+		});
+		process.execArgv = originalArgv;
+
+		t.is(stdout, 'foo');
+		t.is(stderr, '');
+	}
+);
+
+test.serial(
+	'node removes --inspect-brk from nodeOptions when defined by parent process',
+	async t => {
+		const originalArgv = process.execArgv;
+		process.execArgv = ['--inspect-brk', '-e'];
+		const childProc = execa.node('console.log("foo")', {
+			stdout: 'pipe'
+		});
+		process.execArgv = originalArgv;
```

Comment:
The few lines above are repeated across tests: would it be possible to create an helper function instead?